zeppelin-interpreter/pom.xml

```diff
@@ -32,7 +32,7 @@
   <artifactId>zeppelin-interpreter</artifactId>
   <packaging>jar</packaging>
   <version>0.8.0-SNAPSHOT</version>
-  <name>Zeppelin: Interpreter</name>
+  <name>zeppelin-interpreter</name>
```

Comment:
why the change? is this related to ZEPPELIN-3025?

Reply:
Yup! Looks like it's fixed now in master...

Reply:
I merged the master branch and renamed the project name https://github.com/apache/zeppelin/pull/2649/commits/9fc64c1d2a4a8de136e63c224a8b1c392902e488